apps/generic-issuance-client/src/pages/localstorage.ts

```diff
@@ -0,0 +1,99 @@
+import { StytchUIClient } from "@stytch/vanilla-js";
+import validator from "email-validator";
+import { useMemo, useState } from "react";
+import { SESSION_DURATION_MINUTES } from "../constants";
+import { GIContextState } from "../helpers/Context";
+import { DEV_JWT_KEY } from "../helpers/userHooks";
+
+const STATE_KEY = "settings-8";
+
+export function useInitialState(
+  stytch: StytchUIClient | undefined
+): GIContextState {
+  const [devModeAuthToken, setDevModeAuthToken] = useState<string | undefined>(
+    !stytch ? window.localStorage.getItem(DEV_JWT_KEY) ?? undefined : undefined
+  );
+
+  const savedState = useMemo(() => {
+    return loadState();
+  }, []);
+
+  const initialState: GIContextState = {
+    ...savedState,
+    devModeAuthToken,
+    setState: () => {
+      //
```

Comment:
Should this comment explain what's happening here?

Reply:
added a comment